spring-boot-project/spring-boot-dependencies/build.gradle

```diff
@@ -1025,10 +1025,10 @@ bom {
 			]
 		}
 	}
-	library("OpenTelemetry", "1.19.0") {
+	library("OpenTelemetry", "1.23.1-alpha") {
 		group("io.opentelemetry") {
 			imports = [
-				"opentelemetry-bom"
+				"opentelemetry-bom-alpha"
```

Comment:
I'm not sure a stable version of Boot should depend on something that is unstable.
Also, right now upgrading OTel to anything higher than 1.20.1 will break baggage propagation in Micrometer Tracing.